internal/cli/atlas/livemigrations/validation/create.go

```diff
@@ -0,0 +1,121 @@
+			ClusterName:           opts.sourceClusterName,
+			GroupID:               opts.sourceProjectID,
+			Username:              opts.sourceUserName,
+			Password:              opts.sourcePassword,
+			SSL:                   pointy.Bool(opts.sourceSSL),
+			CACertificatePath:     opts.sourceCACertificatePath,
+			ManagedAuthentication: pointy.Bool(opts.sourceManagedAuthentication),
+		},
+		Destination: &mongodbatlas.Destination{
+			ClusterName: opts.destinationClusterName,
+			GroupID:     opts.ConfigProjectID(),
+		},
+		MigrationHosts: opts.migrationHosts,
+		DropEnabled:    pointy.Bool(opts.destinationDropEnabled),
+	}
+}
+
+// mongocli atlas liveMigrations|lm validation create --clusterName clusterName --migrationHosts hosts --sourceClusterName clusterName --sourceProjectId projectId [--sourceSSL] [--sourceCACertificatePath path] [--sourceManagedAuthentication] [--sourceUserName userName] [--sourcePassword password] [--drop] [--projectId projectId].
+func CreateBuilder() *cobra.Command {
+	opts := &CreateOpts{}
+	cmd := &cobra.Command{
+		Use:   "create",
+		Short: "Create one new validation request.",
+		Long:  "Your API Key must have the Organization Owner role to successfully run this command.",
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			return opts.PreRunE(
+				opts.ValidateOrgID,
+				opts.initStore,
+				opts.InitOutput(cmd.OutOrStdout(), createTemplate),
+			)
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return opts.Run()
+		},
+	}
+	cmd.Flags().StringVar(&opts.OrgID, flag.ProjectID, "", usage.ProjectID)
+	cmd.Flags().StringVar(&opts.sourceClusterName, flag.LiveMigrationSourceClusterName, "", usage.LiveMigrationSourceClusterName)
+	cmd.Flags().StringVar(&opts.sourceProjectID, flag.LiveMigrationSourceProjectID, "", usage.LiveMigrationSourceProjectID)
+	cmd.Flags().StringVar(&opts.sourceUserName, flag.LiveMigrationSourceUsername, "", usage.LiveMigrationSourceUsername)
+	cmd.Flags().StringVar(&opts.sourcePassword, flag.LiveMigrationSourcePassword, "", usage.LiveMigrationSourcePassword)
+	cmd.Flags().BoolVar(&opts.sourceSSL, flag.LiveMigrationSourceSSL, false, usage.LiveMigrationSourceSSL)
+	cmd.Flags().StringVar(&opts.sourceCACertificatePath, flag.LiveMigrationSourceCACertificatePath, "", usage.LiveMigrationSourceCACertificatePath)
+	cmd.Flags().BoolVar(&opts.sourceManagedAuthentication, flag.LiveMigrationSourceManagedAuthentication, false, usage.LiveMigrationSourceManagedAuthentication)
+	cmd.Flags().StringVar(&opts.destinationClusterName, flag.LiveMigrationDestinationClusterName, "", usage.LiveMigrationDestinationClusterName)
+	cmd.Flags().StringVar(&opts.ProjectID, flag.LiveMigrationDestinationProjectID, "", usage.LiveMigrationDestinationProjectID)
+	cmd.Flags().StringSliceVar(&opts.migrationHosts, flag.LiveMigrationHost, []string{}, usage.LiveMigrationHostEntries)
+	cmd.Flags().BoolVar(&opts.destinationDropEnabled, flag.LiveMigrationDropCollections, false, usage.LiveMigrationDropCollections)
```

Comment:
[nit] feel free to do this later but I think I left a note on the spec to "confirm" with users when using `--drop`

Reply:
Not in the spec, but adding now and will do 👍